src/index.js

```diff
@@ -0,0 +1,34 @@
+import './style.css';
+
+const todoArr = [
+  { description: 'Get groceries', completed: false, index: 1 },
+  { description: 'Wash the dishes', completed: false, index: 2 },
+  { description: 'Complete todo list project', completed: false, index: 3 },
+];
+
+function displayTodo(arr) {
+  const todoListContainer = document.querySelector('.todo-list');
+  const ul = document.createElement('ul');
+  todoListContainer.appendChild(ul);
+
+  arr.forEach((e) => {
```

Comment:
- [ ] Kindly use `arrow functions` instead of the `function` keyword. Javascript's ES6 brings new syntax and awesome features to make your code more modern and readable. It allows you to write less code and do more. ES6 introduces us to many great features like arrow functions. The arrow function is really awesome and makes your code more readable, more structured, and look modern.